common/formats/cue.cpp

```diff
@@ -249,17 +249,19 @@ void CueSheet::parseFilesContext(const char *line) {
 		int trackNum = atoi(nexttok(s, &s).c_str());
 		String trackType = nexttok(s, &s);
 
-		if (trackNum < 0 || (_currentTrack > 0 && _currentTrack + 1 != trackNum)) {
+		if (trackNum < 0 || (_currentTrack > 0 && _currentTrack + 2 != trackNum)) {
```

Comment:
What is `+ 2`? Please add a comment above

Reply:
Added a comment explaining it.